docker/kernel/httploader/httploader/falcoloader/loader.go

```diff
@@ -0,0 +1,148 @@
+package falcoloader
+
+import "io/ioutil"
+import "io"
+import "os"
+import "crypto/md5"
+import "encoding/hex"
+import "strings"
+import "net/http"
+import "golang.org/x/sys/unix"
+import "log"
+import "unsafe"
+import "compress/gzip"
+import "bytes"
+
+func GetKernelVersion() string {
+	path := "/proc/version"
+
+	file, err := ioutil.ReadFile(path)
+	if err != nil {
+		return "invaildVersion"
+	}
+	version_string := string(file)
```

Comment:
Don't use underscores in Go variable names, see: https://golang.org/doc/effective_go.html#mixed-caps

Also use `gofmt` and `go-lint` on your code to make sure you respect Go coding styles.